docs/source/installation.rst

```diff
@@ -200,168 +200,6 @@ This section outlines the ``sktime`` build requirements. These are required for:
 * installing ``sktime`` from source, e.g., development versions
 * the advanced developer set-up
 
-Build requirements summary
```

Comment:
all the work that went into writing this section 😢

Reply:
good bye, old friend

Reply:
:wave: